aten/src/ATen/native/native_functions.yaml

```diff
@@ -4012,6 +4012,14 @@
   dispatch:
     CompositeImplicitAutograd: narrow_tensor_symint
```

Comment:
You will have to update this to `CompositeExplicitAutograd` and add entry in `derivatives.yaml`.

Reply:
> You will have to update this to `CompositeExplicitAutograd` and add entry in `derivatives.yaml`.

Thank you so much!!!  A follow up question: I got this `AssertionError: narrow_backward appears to be a new operator that aliases its inputs. The codegen expects you to add a corresponding operator to native_functions.yaml: narrow_backward_copy. See Note [view_copy NativeFunctions] for details.` in a [failed job](https://github.com/pytorch/pytorch/actions/runs/4833926807/jobs/8614534323).

Do I need to add `narrow_backward_copy`? Or changing `narrow_tensor_symint` to `CompositeExplicitAutograd` will do.

Reply:
Regarding the error, the `narrow_backward(Tensor grad_output, SymInt[] input_sizes, int dim, Tensor start, SymInt length) -> Tensor(a)` is incorrect because `-> Tensor(a)` implies that you are returning one of the inputs but that is not the case as you allocate a new tensor `grad_input`. Changing the return from `-> Tensor(a)` to `-> Tensor` should mostly do the trick